### ISSUE TITLE
Doesn't work for older versions of Jest

### ISSUE THREAD

**Issue by franklin-ross** [k-g-a/jest-node-exports-resolver#17]
Ahoy, it looks like [this change](https://github.com/k-g-a/jest-node-exports-resolver/commit/900da2e7eac05090b569e5614bf0b6b6838443de) makes `conditions` a requirement, but that stops it working for older versions of Jest that don't provide it. Could we provide a default set of conditions where Jest doesn't pass any to at least try to do the right thing?

**Comment by piranna** [comment]
what changes do you suggest? There's no way to detect the actual platform to provide some safe default values...

**Comment by franklin-ross** [comment]
Yeah, fair enough. I guess I figured a best effort is better than nothing when conditions isn't provided? Otherwise, it straight up doesn't work for 100% of the cases prior to about 27.1, which is pretty recent. And considering this is about providing support for legacy versions of Jest (since it works out of the box for v28) that seems helpful?

Alternatively, is there a way to let users provide the conditions themselves in a config somewhere? I've done something like this as a custom wrapper, but I don't know how many people would dig into the code to work something like this out themselves.

```js
const baseResolver = require('jest-node-exports-resolver')

const defaultConditions = ['require', 'node', 'default']

module.exports = (request, opts) => {
  let options = opts

  if (!options) {
    options = {
      conditions: defaultConditions
    }
  } else if (!options.conditions || options.conditions.length <= 0) {
    options.conditions = defaultConditions
  }

  return baseResolver(request, options)
}
```

**Comment by franklin-ross** [comment]
Or even just a handful of common pre-configured ones and a little documentation, like "for Node based tests, do `'resolver': 'jest-node-exports-resolver/node'` or something. I dunno, just trying to think of ways to make it helpful and useful under more situations, even if it's not perfect or complete.
